docs/blog/newsletter-november-2020.rst

```diff
@@ -0,0 +1,141 @@
+
+.. post:: November 02, 2020
+   :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – November 2020
+#########################################
+
+Hello friends - it’s Beth and the newsletter team here.
+
+And just like that, Prague has been and gone. I spent the days after conference feeling a strange mix of revived and exhausted - in other words, just how I always feel after a Write the Docs conference. Not a bad outcome given the move online! I’m glad it turns out we can still gather and be supported and inspired, even when pandemics conspire against us. Plus, videos of both the main stage talks and lightning talks are now up on `our YouTube playlist <https://www.youtube.com/playlist?list=PLZAeFn6dfHpmRWZJaUwQzsdagW2TtRI2x>`__ - enjoy!
+
+If you’re already looking forward to the next conference, we’ve got you covered: tickets for the `Australia & India conference on 3-4 December </conf/australia/2020/>`__ are on sale. To get a sense of the great stuff you can expect there, check out the `list of speakers </conf/australia/2020/speakers/>`__.
+
+One last thing - the `salary survey </surveys/salary-survey-sep-2020/>`__ is open until 16 November. So if you haven’t filled it out yet, we’d love it if you did.
+
+That’s all the new I've got for you today. Onto the articles from our lovely correspondents!
```

Comment:
```suggestion
That’s all the news I've got for you today. Onto the articles from our lovely correspondents!
```

Reply:
Should it actually be "news" rather than "new" here? Now that I made the suggestion I'm not sure what you intended.

Reply:
Either news or "new updates" there's definitely something missing unless the kids have a new turn of phrase I'm not aware of :D

Reply:
oops. fixed!